src/it/addDevelopers/pom.xml

```diff
@@ -3,7 +3,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>org.codehaus.sonar</groupId>
+  <groupId>org.sonarsource.sonarqube</groupId>
```

Comment:
Just a note: this component is not tied to sonarqube, so the package name is not ideal. 
That being said, really doesn't matter since it's just projects for ITs.

Reply:
i changed the groupId to better fit the plugin